mlflow/store/tracking/abstract_store.py

```diff
@@ -893,3 +893,16 @@ def delete_logged_model(self, model_id: str) -> None:
             model_id: ID of the model to delete.
         """
         raise NotImplementedError(self.__class__.__name__)
+
+    @abstractmethod
+    def link_traces_to_run(self, trace_ids: list[str], run_id: str) -> None:
```

Comment:
We'll want to mark this as not-supported in FileStore. I created decorators to help with this handling (providing helpful messages to users about if they hit these APIs when using FileStore how they can setup a simple sqllite tracking server to enable the usage of APIs that only work with a SQL backend.

https://github.com/mlflow/mlflow/pull/16992/files#diff-b18433ac99f116b770ce89cdedfefaa3a44f11d78067ef848783045a41d5b78eR231-R253